utils/dchemigen/get_image_jpg.c

```diff
@@ -0,0 +1,124 @@
+
```

Comment:
Needs a KOS copyright header.

Reply:
This is missing header where I lifted it from (dcbumpgen).